### PR TITLE
18 domains, checked manually, most of them are in Alexa top 100 rank for Spain

### PR DESCRIPTION
**Pull request by i-rme**
18 domains, checked manually, most of them are in Alexa top 100 rank for Spain

### CODE REVIEW

**Comment by i-rme** [comment]
Try to fix Travis error, (duplicated ingdirect.es.xml file)

**Comment by i-rme** [comment]
Another duplicate (adf.ly.xml)

**Comment by i-rme** [comment]
.xml extension added in bandaancha.eu

**Comment by i-rme** [comment]
Deleted amazon.es and Boe.es because they were duplicate.

I hope this is the last fail

File: src/chrome/content/rules/1and1.es.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<ruleset name="1and1.es">
+  <target host="1and1.es" />
```
Comment (loveithateit) [inline src/chrome/content/rules/1and1.es.xml]: Cert doesn't match ^1and1.es, please rewrite to www.1and1.es

File: src/chrome/content/rules/bet365.es.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<ruleset name="bet365.es">
+  <target host="bet365.es" />
```
Comment (loveithateit) [inline src/chrome/content/rules/bet365.es.xml]: Connections are rejected for ^s://bet365.es, please rewrite to www.bet365.es

File: src/chrome/content/rules/pccomponentes.com.xml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<ruleset name="pccomponentes.com">
```
Comment (loveithateit) [inline src/chrome/content/rules/pccomponentes.com.xml]: There's mixed css, so please add platform="mixedcontent"

File: src/chrome/content/rules/www.elcorteingles.es.xml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+<ruleset name="elcorteingles.es">
+  <target host="elcorteingles.es" />
```
Comment (loveithateit) [inline src/chrome/content/rules/www.elcorteingles.es.xml]: Cert doesn't match ^elcorteingles.es, please rewrite to www.elcorteingles.es

File: src/chrome/content/rules/www.movistar.es.xml [hunk 5]
```diff
@@ -0,0 +1,6 @@
+<ruleset name="movistar.es">
+  <target host="movistar.es" />
```
Comment (loveithateit) [inline src/chrome/content/rules/www.movistar.es.xml]: Cert doesn't match ^movistar.es, please rewrite to www.movistar.es

**Pull request merged**